align.py

```diff
@@ -0,0 +1,49 @@
+import argparse
```

Comment:
You provide repository with Apache 2.0, but don't follow its' recommendation on how to apply -- https://github.com/MobileRoboticsSkoltech/bag-extractor/blob/65e9c65d8b4d45dbd43f5674cc24afe5c9420466/LICENSE#L178 , so it's preferably to add headers to every file excepting, maybe, some config things.

Reply:
[Apache FAQ](https://infra.apache.org/apply-license.html#copy-per-file) says: "You only need to add one full copy of the license per distribution."

It seems this is optional, but maybe we should use these headers to make licensing more apparent when viewing a single file?